compiler/src/dotty/tools/dotc/reporting/messages.scala

```diff
@@ -2743,8 +2744,15 @@ class MissingImplicitArgument(
         // show all available additional info
         def hiddenImplicitNote(s: SearchSuccess) =
           i"\n\nNote: ${s.ref.symbol.showLocated} was not considered because it was not imported with `import given`."
+        def noChainConversionsNote(s: Iterable[TermRef]): Option[String] =
+          Option.when(s.nonEmpty)(
+              i"\n\nNote: Chaining implicit conversions are not allowed in Scala. " +
+              i"The following conversions are in scope:${s.map(g => s"\n  - ${g.symbol.showDcl}").mkString}" +
+              i"\nIf there is an implicit `Conversion[A, ${pt.show}]` and an implicit `A` in scope, you can explicitly pass the argument, e.g. `(using summon[A])`"
```

Comment:
Is this supposed to be a generic tip or a complete instruction telling a user how to fix their code in a particular situation? This message seems to be something in between, which is misleading. That's because `pt` is substituted for the actual searched type here but `A` is just some example type unrelated to the user's code and it might even clash with the actual searched type if `pt` was `A`. Also if a method has multiple parameters in a using clause, one has to pass none or all of them, eg. `(using summon[A], summon[B], summon[C])`.

Reply:
Ah, yeah, it might be better to turn it into a completely generic tip, since the message doesn't know about the other arguments.